CIScripts/Build.ps1

```diff
@@ -41,7 +41,7 @@ foreach ($Directory in $Directories) {
 }
 
 try {
-    Invoke-CnmPluginBuild -DriverSrcPath $Env:DRIVER_SRC_PATH `
+    Invoke-CnmPluginBuild -PluginSrcPath $Env:PLUGIN_SRC_PATH `
```

Comment:
As we talked IRL, maybe change this environment variable to CNM_PLUGIN_SRC_PATH?